Modules/_cursesmodule.c

```diff
@@ -4586,17 +4570,24 @@ make_ncurses_version(PyTypeObject *type)
     if (ncurses_version == NULL) {
         return NULL;
     }
-
+    const char *str = curses_version();
+    unsigned long major = 0, minor = 0, patch = 0;
+    if (!str || sscanf(str, "%*[^0-9]%lu.%lu.%lu", &major, &minor, &patch) < 3) {
```

Comment:
I dislike having to parse a string to get a version number while we have directly needed values as NCURSES_VERSION_xxx constants.

Would it be possible to only use this code on macOS, or maybe even only on macOS when we know that NCURSES_VERSION_xxx constants can be wrong?

Please add a comment why you don't trust NCURSES_VERSION_xxx constants. Add a reference to the issue, and maybe include an example of inconsistent versions (str vs integers).

Reply:
@serhiy-storchaka: do you think that it's worth it to parse a string to get the version of the dynamic library, rather than the version used to build Python?

Reply:
Is this change related to this issue?